webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/Compressor.scala

```diff
@@ -256,3 +260,25 @@ class BloscCompressor(val properties: Map[String, CompressionSetting]) extends C
     bs
   }
 }
+
+class JpegCompressor() extends Compressor {
+
+  override def getId = "jpeg"
+
+  override def toString: String = getId
+
+  @throws[IOException]
+  override def compress(is: InputStream, os: OutputStream): Unit =
+    throw new NotImplementedError()
```

Comment:
```suggestion
  override def compress(is: InputStream, os: OutputStream): Unit = ???
```

(that’s the same in scala)